rj_gameplay/stp/utils/constants.py

```diff
@@ -46,6 +49,32 @@ class BallConstants:
 
 class EvaluationConstants:
     SLOW_THRESHOLD = 1
+    SETTLE_BALL_SPEED_THRESHOLD = 1.0
+    INVALID_COST = sys.maxsize
+
+
+class KickConstants:
+    KICK_DOT_THRESHOLD = 0.4
+    KICK_BALL_SPEED_THRESHOLD = 0.9
+    MAX_KICK_SPEED = 5.0
+
+
+class GoalieConstants:
+    MIN_WALL_RAD = 0
+    GOALIE_PCT_TO_BALL = 0.15
+    DIST_TO_FAST_KICK = 7
+
+
+class SituationConstants:
```

Comment:
Hey, these constants + StrikerConstants below are named well, but they don't seem to have been actually used in gameplay? I can't find EFF_BLOCK_WIDTH referenced anywhere else, for instance, but I believe from the name it should be in role/striker.py